nvflare/app_common/workflows/broadcast_operator.py

```diff
@@ -55,15 +56,19 @@ def multicasts_and_wait(
         task_name: str,
         task_inputs: Dict[str, Shareable],
         abort_signal: Signal = None,
+        task_check_period: int = 0.5,
     ) -> Dict[str, DXO]:
 
         tasks: Dict[str, Task] = self.get_tasks(task_name, task_inputs)
         for client_name in tasks:
-            self.controller.broadcast(task=tasks[client_name], fl_ctx=self.fl_ctx, targets=[client_name])
+            self.controller.send(task=tasks[client_name], fl_ctx=self.fl_ctx, targets=[client_name])
```

Comment:
It's a bad idea to keep fl_ctx in self. Instead, the method multicasts_and_wait should have a fl_ctx passed to it. The caller needs to provide fl_ctx.